src/collections/Grid/GridRow.js

```diff
@@ -10,14 +10,14 @@ export default class GridRow extends Component {
 
   static _meta = {
     library: META.library.semanticUI,
-    name: 'Row',
+    name: 'GridRow',
     type: META.type.collection,
     parent: 'Grid',
   };
 
   render() {
     const classes = classNames(
-      'sd-row',
+      'sd-grid-row',
```

Comment:
This is something to think about for sure.  Since we are able to test for them, I believe should be able to generate them.  Will give it more thought.